src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/README.md

```diff
@@ -0,0 +1,26 @@
+# Common Language Server Protocol Framework (CLaSP)
+
+CLaSP was created to ease the creation and maintenance of [Language Server](https://microsoft.github.io/language-server-protocol/) implementations by sharing our hard-won knowledge, while leaving you the flexiblity to create the server you need.
+
+## Getting Started with CLaSP
```

Comment:
An alternative format for this document you could consider - build up to the example project step by step in this, putting in code examples as you go.  It might make it easier to grok when each step builds on each other with a visual of what the code actually looks like.

Reply:
I updated the document to include links to the Example implementation at relevant portions of the getting started. I didn't want to include copies of the code for fear they would become stale and crowd the page, hopefully this is a good compromise.

I also included links for Roslyn and Razor to the relevant portions with call-outs for areas I thought might be particularly educational.